File: EvaluationSpaceAPI/Entities/Question.cs
using System;
using System.Collections.Generic;

namespace EvaluationSpaceAPI.Entities;

public partial class Question
{
    public Guid Id { get; set; }

    public Guid IdQuiz { get; set; }

    public int IdQuestionType { get; set; }

    public string QuestionText { get; set; } = null!;

    public int Score { get; set; }

    public virtual ICollection<Answer> Answers { get; } = new List<Answer>();

    public virtual QuestionType IdQuestionTypeNavigation { get; set; } = null!;

    public virtual Quiz IdQuizNavigation { get; set; } = null!;
}

File: EvaluationSpaceUI/src/app/models/question.ts
import {Answer} from "./answer";

export class Question {
  questionText: string;
  score: number;
  idQuestionType: number;
  answers: Answer[];

  constructor(questionText: string, score: number, idQuestionType: number, answers: Answer[]) {
    this.questionText = questionText;
    this.score = score;
    this.idQuestionType = idQuestionType;
    this.answers = answers;
  }
}
